geowebcache/core/src/main/resources/org/geowebcache/config/geowebcache.xsd

```diff
@@ -614,8 +619,6 @@
           <xs:element name="wmsVersion" minOccurs="0">
             <xs:annotation>
               <xs:documentation xml:lang="en">
-                The VERSION parameter sent to the WMS backend.
-                The default is 1.1.1

```

Comment:
Not sure what happened on that one, certainly not deliberate.  Sorry missed it in the commit.